docs/formats/zst.md

```diff
@@ -190,128 +117,132 @@ independently of each other, similar to zng compression framing.
 ### The Reassembly Section
 
 The reassembly section provides the information needed to reconstruct
-column streams from segments, and in turn, to reconstruct the original zng rows
-from column streams, i.e., to map columns back to rows.
+column streams from segments, and in turn, to reconstruct the original Zed values
+from column streams, i.e., to map columns back to composite values.
 
 > Of course, the reassembly section also provides the ability to extract just subsets of columns
 > to be read and searched efficiently without ever needing to reconstruct
 > the original rows.  How performant this is all done is up to any particular
-> zst implementation.
+> ZST implementation.
 
 > Also, the reassembly section is in generally vastly smaller than the data section
 > so the goal here isn't to express information in cute and obscure compact forms
 > but rather to represent data in easy-to-digest, programmer-friendly form that
-> leverages zng.
+> leverages ZNG.
 
-The reassembly section is a zng stream.  Unlike parquet,
+The reassembly section is a ZNG stream.  Unlike Parquet,
 which uses an externally described schema
 (via [Thrift](https://thrift.apache.org/)) to describe
-analogous data structures, we simply reuse zng here.
+analogous data structures, we simply reuse ZNG here.
 
-> So, we are using zng to encode zng in column format.  Yo dawg.
+#### The Super Types
```

Comment:
I would consider dropping the "The" here
```suggestion
#### Super Types
```

Reply:
Same comment as above.